src/bin/qml/pages/UsersPage.qml

```diff
@@ -108,7 +108,7 @@ Page {
 
         StatusPlaceholder {
             model: userModel
-            errorMessage: (model.status === Model.Idle && model.count === 0) ? qsTr("No friends") : ""
+            errorMessage: (model.status === Model.Idle && model.count === 0) ? qsTr("No follower") : ""
```

Comment:
follower or followings 
I was not able to see it in the UI where it pops up :-S